frontend/package.json

```diff
@@ -37,5 +43,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "axios-mock-adapter": "^1.19.0"
```

Comment:
Ezt nem devDependencybe raknám egyelőre hanem simába, mert a production build-nek is kell, viszont ott nem lesz meg.

Reply:
átraktam dependencies-be.